.gitignore

```diff
@@ -5,3 +5,4 @@ django_picklefield.egg-info/*
 build/
 .coverage
 .tox
+.DS_Store
```

Comment:
You should add this line to your own global `.gitignore` file as it has nothing to do with this project; it's specific to your OS.

```suggestion
```